src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -20,6 +20,7 @@
     <DebugType>embedded</DebugType>
     <!-- See https://github.com/Azure/azure-functions-durable-extension/issues/1433 -->
     <NoWarn>NU5125;SA0001</NoWarn>
+    <DefineConstants>$(DefineConstants);FUNCTIONS_V2_OR_GREATER;FUNCTIONS_V3_OR_GREATER</DefineConstants>
```

Comment:
Yeah I think you are right. I did this because Jacob's sample draft PR in our thread added this. And I want to confirm this here. 

But if we will support Functions v4 only, I can delete all the #if blocks.